### DISCUSSION TITLE
How to drag connection line to other position?

### DISCUSSION THREAD

**Discussion by flashvnn** [bcakmakoglu/vue-flow#608] [Q&A] [answered]
I want to drag connection line between two node, is there way to implement it?

![2023-01-16__13-22-38](https://user-images.githubusercontent.com/1784547/212611159-81d8ecc8-de97-40cc-9bef-b32a6329d80c.png)

**Comment by bcakmakoglu** [comment, marked as answer]
Nope, that's not possible with the current step/smoothstep edges. You'd have to implement your own solution for that, sorry.

**Reply by flashvnn** [reply]
Thank for your response, other question where i can change the way draw the connection line near to Content 2, not draw in the center of Content 1 and Content 2

![2023-01-16__15-40-10](https://user-images.githubusercontent.com/1784547/212634026-893f8ae7-feb7-41a6-99ee-27e1f7b0356e.png)

**Reply by bcakmakoglu** [reply]
The issue is by default it will always draw it in the center.
What you can do is use a custom edge and use `getSmoothStepPath` and pass a `centerX` and `centerY` value to adjust where the center of the edge is supposed to be.
That requires you to know what XY position the center is supposed to be obviously, so you'd have to calculate the center yourself and move it further to the right/left.

I will add an export for `getSimpleEdgeCenter` in the next minor release (`1.13.0`) which you can use the get the actual center XY values and adjust those (by adding a couple px or something) but currently, you'll have to calculate it yourself :/
